### PR TITLE
46 products database

### PR DESCRIPTION
**Pull request by jsjlewis96**

### CODE REVIEW

**Comment by Mattihew** [comment]
have you got the SQL for the database?

File: YHP_website/src/main/java/storage/ProductCache.java [hunk 1]
```diff
@@ -0,0 +1,115 @@
...
+	}
+	
+	private class ProductResultsSetHandler extends AbstractListHandler<Product>
+	{
+		private ResultSet rs;
+		
+		private String getValue(final Column column) throws SQLException
+		{
+			return this.rs.getString(column.toString());
+		}
```
Comment (Mattihew) [inline YHP_website/src/main/java/storage/ProductCache.java]: I like this, much tidyer than the usercache one. But if you create another method that takes a `Column` and returns an int would save going from int -> String -> int.

Comment (Mattihew) [inline YHP_website/src/main/java/storage/ProductCache.java]: I like this, much tidyer than the usercache one. But if you create another method that takes a `Column` and returns an int would save going from int -> String -> int.

**Comment by jsjlewis96** [comment]
SQL is in wiki -> database structures

**Comment by Mattihew** [comment]
uh... pretty sure that price should be int. not character varing

**Comment by jsjlewis96** [comment]
good point, didn't realise at the stime there was a getPriceInPennies

**Comment by Mattihew** [comment]
Looks good we'll try it out when we get the ui hooked up to it

**Pull request merged**